crowbar_framework/config/routes.rb

```diff
@@ -62,13 +62,11 @@
 
   #support paths
   get "utils(.:format)", controller: "support", action: "index", as: "utils"
-  get "utils/files/:id(.:format)", controller: "support", action: "destroy", constraints: { id: /[^\/]+/ }, as: "utils_files"
+  delete "utils/files/:id(.:format)", controller: "support", action: "destroy", constraints: { id: /[^\/]+/ }, as: "utils_files"
```

Comment:
Metrics/LineLength: Line is too long. [128/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)